src/Frontend/Core.hs

```diff
@@ -401,33 +464,36 @@ form formHandler = getH :<|> postH
 -- | Wrap anything that has 'ToHtml' and wrap it in an HTML body with complete page.
 data Frame body
     = Frame { _frameUser :: User, _frameBody :: body, _frameMessages :: [StatusMessage] }
-    | PublicFrame               { _frameBody :: body, _frameMessages :: [StatusMessage] }
+    | PublicFrame               { _frameBody :: body, _frameMessages :: [StatusMessage] } -- TODO remove messages
```

Comment:
I'm not really sure it makes sense to display messages to not logged in users. If the user is not logged in then the user state is empty therefor the list of messages is empty therefore we don't need to have it here.